integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/gitservice/GitService.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.gitservice;
+
+import org.wso2.choreo.connect.tests.context.CCTestException;
+import org.wso2.choreo.connect.tests.util.Utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+
+public class GitService {
+
+    public static void addGitServiceToDockerCompose(String dockerComposePath, String gitServiceFile) throws IOException, CCTestException {
+        File targetClassesDir = new File(GitService.class.getProtectionDomain().getCodeSource().
+                getLocation().getPath());
+        String targetDir = targetClassesDir.getParentFile().toString();
+        String gitService = GitService.class.getClassLoader()
+                .getResource("dockerCompose/" + gitServiceFile).getPath();
+        // Input files
+        List<Path> inputs = Arrays.asList(
+                Paths.get(dockerComposePath),
+                Paths.get(gitService)
+        );
```

Comment:
Shall we keep this declaration within a single line ?

Reply:
Fixed with 763b649